applications/tari_indexer/src/config.rs

```diff
@@ -83,6 +83,8 @@ pub struct IndexerConfig {
     pub graphql_address: Option<SocketAddr>,
     /// The address of the HTTP UI
     pub http_ui_address: Option<SocketAddr>,
+    /// The jrpc address where the UI should connect
+    pub jrpc_for_ui_address: Option<String>,
```

Comment:
You can use 
```suggestion
    pub jrpc_for_ui_address: Option<SocketAddr>,
```

Reply:
I can't because there will be no real ip in some case, it will be a domain name.